README.md

```diff
@@ -159,42 +157,42 @@ You will start the timer when you release the first cell, you can play with your
 
 To activate the keyboard controller mode, press either **W**, **A**, **S** or **D**.
 
-* **Escape**: Exit keyboard controller mode;
```

Comment:
Removing the semicolumn is ok, but the text you can keep at it was before (for all keys)